contracts/src/interfaces/ITeleporterTokenBridge.sol

```diff
@@ -32,6 +32,97 @@ struct SendTokensInput {
     uint256 requiredGasLimit;
 }
 
+/**
+ * @notice Parameters for briding tokens to another chain and calling a contract on that chain
+ * @param destinationBlockchainID blockchainID of the destination
+ * @param destinationBridgeAddress address of the destination token bridge instance
+ * @param recipientContract the contract on the destination chain that will be called
+ * @param recipientPayload the payload that will be provided to the recipient contract on the destination chain
+ * @param requiredGasLimit the required amount of gas needed to deliver the message on its destination chain,
+ *                         including token operations and the call the recipient contract.
+ * @param recipientGasLimit the amount of gas that will provided to the recipient contract on the destination chain,
+ *                          which must be less than the requiredGasLimit of the message as a whole.
+ * @param fallbackRecipient address where the bridged tokens are sent if the call to the recipient contract fails.
+ * @param primaryFee amount of tokens to pay for Teleporter fee on the source chain
+ * @param secondaryFee amount of tokens to pay for Teleporter fee if a multihop is needed
+ */
+struct SendAndCallInput {
+    bytes32 destinationBlockchainID;
```

Comment:
wondering if we should replace this with a `SendTokensinput` here. Some of the naming might be as clear though

Reply:
Yeah I decided against it since solidity doesn't supported embedded struct composition similar to Golang. I think it'd be more confusing/long winded to have refer to elements such as `input.SendTokensInput.destinationBlockchainID` rather than having the common fields duplicated across both types.